src/certifier.mak

```diff
@@ -68,18 +76,34 @@ ifdef ENABLE_SEV
 dobj += $(O)/sev_support.o $(O)/sev_report.o
 endif
 
-all:	$(CL)/certifier.a
+# Objs needed to build Certifer Framework shared lib for use by Python module
+cfsl_dobj := $(dobj) $(O)/certifier_framework_wrap.o
+
+CERTIFIER_LIB = certifier.a
+CERTIFIER_SHARED_LIB = libcertifier_framework.so
+
+# RESOLVE: We are invoking shared lib target first, which means
+# CFLAGS_PIC = -fpic will be included in CFLAGS for all objects.
+# Rearrange so that for normal builds we don't include CFLAGS_PIC = -fpic
+# and only re-build all required objects with -fpic for shared library's build.
```

Comment:
RESOLVE: Need to think how to fix this issue.

Reply:
Resolved. Added `sharedlib` as new target and changed build.yml to invoke `make` with `--always-make` flag.